presto-main/src/test/java/com/facebook/presto/cost/TestScalarStatsCalculator.java

```diff
@@ -87,4 +92,38 @@ public void testLiteral()
                 .highValueUnknown()
                 .nullsFraction(1.0);
     }
+
```

Comment:
move assertion methods to the bottom (below tests) https://github.com/Teradata/presto/pull/573/commits/0b6441bfaf71a665132240c185154ad88b800599#diff-39a46d14265f897e3ccaf3e049e987e0R53